Delta/Emulation/GameViewController.swift

```diff
@@ -832,6 +837,12 @@ extension GameViewController: SaveStatesViewControllerDelegate
         saveState.modifiedDate = Date()
         saveState.coreIdentifier = self.emulatorCore?.deltaCore.identifier
         
+        if ExperimentalFeatures.shared.toastNotifications.stateSaveEnabled,
+           saveState.type != .auto
+        {
+            self.presentExperimentalToastView(NSLocalizedString("Game State Saved", comment: ""))
```

Comment:
```suggestion
            self.presentExperimentalToastView(NSLocalizedString("Save State Saved", comment: ""))
```

OR

```suggestion
            self.presentExperimentalToastView(NSLocalizedString("Saved Save State", comment: ""))
```

[Request] I know it's redundant...but for consistency with rest of the app, we should call them Save States not Game States for now. (Open to renaming in the future though)